### DISCUSSION TITLE
Request to make the PCAP capture page with jobs specific to signed in user.

### DISCUSSION THREAD

**Discussion by Hops-AF** [Security-Onion-Solutions/securityonion#2768] [Ideas] [answered]
I'm working with a team to create a few training options and I realized that it doesn't matter what user I'm logged in as on I can see all of the pcap jobs. While possibly beneficial it also could result in confusion. Would it be possible to make this a per user view vs a global view?

**Comment by TOoSmOotH** [comment, marked as answer]
#2775 is tracking this
